src/Mvc/Mvc.Core/src/ModelBinding/Metadata/DefaultMetadataDetails.cs

```diff
@@ -54,6 +54,11 @@ public DefaultMetadataDetails(ModelMetadataIdentity key, ModelAttributes attribu
         /// </summary>
         public ModelMetadata[] Properties { get; set; }
 
+        /// <summary>
+        /// Gets or sets the <see cref="ModelMetadata"/> entries for constructor parameters.
+        /// </summary>
+        public ModelMetadata[] ConstructorParameters { get; set; }
```

Comment:
Suggest `BoundConstructorParameters` to align w/ my suggestion in src/Mvc/Mvc.Abstractions/src/ModelBinding/ModelMetadata.cs